test/acceptance/features/bindAppToServiceUsingSecret.feature

```diff
@@ -394,3 +394,80 @@ Feature: Bind values from a config map referred in backing service resource
         Then Service Binding "ssd-2" is ready
         And The application env var "BACKEND_USERNAME" has value "AzureDiamond"
         And The application env var "BACKEND_PASSWORD" has value "hunter2"
+
+    Scenario: Verify pulling data from a secret living in another namespace
```

Comment:
the ticket is about verifying, but the scenario name itself should describe the intention, i.e. how system should work from user perspective. Thus:

```
Inject into app all keys from a secret existing in a namespace different from the service resource
```

Reply:
Done.